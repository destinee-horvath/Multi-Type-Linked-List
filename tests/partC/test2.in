NEW 3
b
c
d
NEW 3
1
2
3
NEW 4
start
{0}
{1}
end
VIEW ALL
VIEW 0
VIEW 1
VIEW 2
VIEW-NESTED 2
TYPE 0
TYPE 1
TYPE 2
