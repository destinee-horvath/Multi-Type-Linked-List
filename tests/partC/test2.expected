List 0: b -> c -> d
List 1: 1 -> 2 -> 3
Nested 2: start -> {List 0} -> {List 1} -> end
Number of lists: 3
List 0
List 1
Nested 2
b -> c -> d
1 -> 2 -> 3
start -> {List 0} -> {List 1} -> end
start -> {b -> c -> d} -> {1 -> 2 -> 3} -> end
char -> char -> char
int -> int -> int
string -> reference -> reference -> string
